### ISSUE TITLE
Bug in XP progress chart

### ISSUE THREAD

**Issue by tylerecouture** [bytedeck/bytedeck#1657]
Not sure what's causing this bug star wars-like thing to appear in the legend. Dependancy?  jscharts?

![image](https://github.com/user-attachments/assets/de362a0c-4058-4c32-a279-d288e9def904)

**Comment by 9921622** [comment]
@tylerecouture im not getting this bug

tried rebuilding containers, force reinstalled `requirements.txt`, tried on a separate tenant.

accessing chart using a `staff` account
![image](https://github.com/user-attachments/assets/26c8c898-9114-4152-9f1b-6656250f9187)

accessing chart using a student account
![image](https://github.com/user-attachments/assets/ad0d523f-dac5-42d4-91fd-b38925b85676)

**Comment by tylerecouture** [comment]
@9921622 Have you tried different browsers?

**Comment by tylerecouture** [comment]
I don't see it in Chromium, but I do see it in Firefox

**Comment by 9921622** [comment]
@tylerecouture yep. Used google chrome for the staff one, and brave for the student one. Both use chromium so thats probably why

**Comment by tylerecouture** [comment]
It's gone in Firefox now too.  Fixed itself =)
